packages/hubspot_client/src/hubspot_client.ts

```diff
@@ -38,6 +38,7 @@ const ALLOWED_USER_FIELDS = {
     billing_email__c: _.isString,
 
     lifecyclestage: _.isString,
+    apify_kanban_link_: _.isString,
```

Comment:
is the trailing underscore intentional?

Reply:
Unfortunately yes 😄  The field is named `Apify kanban link:` in Hubspot - they seem to replace non-alphanum chars with underscores...